src/test/kotlin/org/rust/ide/ssr/RsSSRTestBase.kt

```diff
@@ -9,16 +9,20 @@ import com.intellij.structuralsearch.inspection.SSBasedInspection
 import com.intellij.structuralsearch.inspection.StructuralSearchProfileActionProvider
 import com.intellij.structuralsearch.plugin.ui.SearchConfiguration
 import org.intellij.lang.annotations.Language
+import org.rust.ide.experiments.RsExperiments
 import org.rust.ide.inspections.RsInspectionsTestBase
 import org.rust.lang.RsFileType
+import org.rust.openapiext.runWithEnabledFeatures
 
 abstract class RsSSRTestBase : RsInspectionsTestBase(SSBasedInspection::class) {
     protected fun doTest(@Language("Rust") code: String, pattern: String) {
-        val configuration = SearchConfiguration().apply {
-            matchOptions.setFileType(RsFileType)
-            matchOptions.fillSearchCriteria(pattern.trimIndent())
+        runWithEnabledFeatures(RsExperiments.SSR) {
+            val configuration = SearchConfiguration().apply {
+                matchOptions.setFileType(RsFileType)
+                matchOptions.fillSearchCriteria(pattern.trimIndent())
+            }
+            StructuralSearchProfileActionProvider.createNewInspection(configuration, project)
+            checkWarnings(code)
```

Comment:
Btw we have `@WithExperimentalFeatures` annotation that can be applied to a test class, it could be nicer to use it here